### PR TITLE
Add Dable ad functionality

### PR DESCRIPTION
**Pull request by goonoo**
Support case that page has no item id.

### CODE REVIEW

File: ads/dable.js [hunk 1]
```diff
@@ -42,13 +42,16 @@ export function dable(global, data) {
...
   const itemId = data['itemId'] || '';
+  const opts = {};
 
   if (itemId) {
     global.dable('sendLog', 'view', {id: itemId});
+  } else {
+    opts.ignoreItems = true;
   }
 
   // call render widget
```
Comment (cathyxz) [inline ads/dable.js]: Do you have documentation on this that could benefit from noting this update? =)

Comment (mctenshi) [reply ads/dable.js]: Oh, thanks for reply.

Before this update, page without `itemId` render dable widget within 3 seconds delay. After this update, the delay will be removed.

**Comment by zhouyx** [comment]
@mctenshi This change looks good. However when I try to test with your demo ad, I could not see the ad load. Get `Dable DEBUG: Unknown SERVICE: localhost`

Since I couldn't verify that your ad example works after the change. I'll double check with you that you feel the change is safe enough to be merged. 

BTW: could you provide a fake ad, that can be tested with our local server? Just for demo and test purpose. Thanks.

**Comment by zhouyx** [comment]
@mctenshi Friendly ping : )

**Comment by mctenshi** [comment]
I added domain `localhost` for test. Now you can see our demo with your local server.

Thanks.

**Comment by zhouyx** [comment]
@mctenshi Now the example always return me a no-fill ad. Could you provide a fake ad for test purpose. Thanks.

**Comment by mctenshi** [comment]
Updated dable ad example for test purpose. Thanks.

**Comment by zhouyx** [comment]
@goonoo LGTM! I've restarted the percy test. In case it still fails, please pull and rebase to latest master following https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#pull-the-latest-changes-from-the-amphtml-repository

Thanks.

**Pull request merged**